index.html

```diff
@@ -1,3 +1,29 @@
-<html>
-  <h1>hello main</h1>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+        <h1>
+            My Logo
+        </h1>
+        <a href="#">Portfolio</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+    </header>
+    <section class="mainsection">
+        <div class="text-section">
+            <p class="text1">I’m Bunbee <br>Glad to see you!</p>
+            <p class="text2">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <a href="#">LET'S CONNECT</a>
+        </div>
+    </section>
+</body>
```

Comment:
- [ ] You are missing the social media icons as suggested on the project requirements.

The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag)